src/GraphQL.Tests/Execution/MutationTests.cs

```diff
@@ -240,7 +240,7 @@ public void evaluates_mutations_correctly_in_the_presense_of_a_failed_mutation()
                 }";
 
             var result = AssertQueryWithErrors(query, expected, root: new Root(6), expectedErrorCount: 2);
-            result.Errors.First().InnerException.Message.ShouldBe("Cannot change the number");
+            result.Errors.First().InnerException.InnerException.Message.ShouldBe("Cannot change the number");
```

Comment:
@joemcbride Yes, can do – reuse the logic [here](https://github.com/graphql-dotnet/graphql-dotnet/pull/221/files#diff-9ee075dfc3429298a3c5de0c1b53bef2L325) to report the `task.Exception` then you mean?